### ISSUE TITLE
By the gods, PLEASE stop Discord from displaying you're using this.

### ISSUE THREAD

**Issue by BlepBlep** [Seklfreak/discord-image-downloader-go#112]
It literally forces you online (even if you set yourself offline) and puts your status as your downloading images. Kind of defeats the purpose of having a bot to do this, or using it on servers with NSFW channels and whatnot (I'll admit it lol). Get ridiculed every time this bot is pulled out. Please stop it from doing this, or point me to one that doesn't.

**Comment by Jake-Watkins** [comment]
I'd recommend using a separate discord account exclusively for this, even
if just because user bots are not allowed and you run the risk of having
your actual discord account banned. Plus you also can then use the history
command to download channel histories.

Unless the situation is that you are using this bot in very small server,
where even if someone is not a friend with you, they will still see an
account with this status, just by looking at the right side. Is that what
you mean?

**Comment by BlepBlep** [comment]
Yeah but it's also for DM downloads from friends and groups. Stuff the second account can't join.

**Comment by Resak** [comment]
> 
> 
> It literally forces you online (even if you set yourself offline) and puts your status as your downloading images. Kind of defeats the purpose of having a bot to do this, or using it on servers with NSFW channels and whatnot (I'll admit it lol). Really annoying to get ridiculed every time this bot is pulled out. Please stop it from doing this, or point me to one that doesn't.

I second this, or if you could point me to the part of the program which displays this, so I can try to change it for myself.

**Comment by Resak** [comment]
> 
> 
> It literally forces you online (even if you set yourself offline) and puts your status as your downloading images. Kind of defeats the purpose of having a bot to do this, or using it on servers with NSFW channels and whatnot (I'll admit it lol). Really annoying to get ridiculed every time this bot is pulled out. Please stop it from doing this, or point me to one that doesn't.

I second this. 
Part of the program, where it displays the status message would be awesome as well, I could try to change it myself then.

**Comment by Kakkela** [comment]
This feature exist so you have to be transparent with your bot downloads. 
Not everyone will look at you doing it nicely and it's not the bots author job to hide it for you so you don't need to be transparent about it.

There has been people before you and if you search through previous issues you will find what you are looking for.
